alterclientquotas_test.go

```diff
@@ -0,0 +1,104 @@
+package kafka
+
+import (
+	"context"
+	"testing"
+
+	ktesting "github.com/segmentio/kafka-go/testing"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClientAlterClientQuotas(t *testing.T) {
+	// Added in Version 2.6.0 https://issues.apache.org/jira/browse/KAFKA-7740
+	if !ktesting.KafkaIsAtLeast("2.6.0") {
+		return
+	}
+
+	const (
+		entityType = "client-id"
+		entityName = "my-client-id"
+		key        = "producer_byte_rate"
+		value      = 500000.0
+	)
+
+	client, shutdown := newLocalClient()
+	defer shutdown()
+
+	expectedAlterResp := AlterClientQuotasResponse{
```

Comment:
Maybe it's the standard practice to define the expected value early but it feels more logical to me just before the assert (and after the err check)